packages/typespec-python/test/specification/apitest/limit/main.tsp

```diff
@@ -0,0 +1,22 @@
+import "@typespec/rest";
+import "@typespec/http";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+
+@service({
+  title: "API Test",
+  version: "2023-10-18",
+})
+@server("https://example.com", "Single server endpoint")
+namespace Apitest.Limit;
+
+@route("/apitest/limit")
```

Comment:
@msyyc we don't want to add any more tests here, this should all go in cadl-ranch. I'll work on this today

Reply:
I have removed the testcase